editor/js/libs/ui.three.js

```diff
@@ -571,16 +571,11 @@ class UIPoints extends UISpan {
 
 	clear() {
 
-		for ( let i = 0; i < this.pointsUI.length; ++ i ) {
```

Comment:
Instead of using a while loop, can you also reverse the order in the `for` loop instead. 
```js
for ( let i = this.pointsUI.length - 1; i >= 0; -- i ) {
```

Reply:
ok.